src/Messenger/Middleware/ImageStatePreparedStamp.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusImagePlugin\Messenger\Middleware;
```

Comment:
We already have a `Message` namespace. Should we maybe put it inside that? Or what do other bundles do?